examples/Flight2/Flight2.ino

```diff
@@ -0,0 +1,920 @@
+/**
```

Comment:
Seems some formatting are not correcT.
Use the Arduino IDE 1.8.19 to format it: ctrl + T.
It will format it properly.